setup.py

```diff
@@ -65,7 +65,14 @@ def load_requirements(*requirements_paths):
         'workbench',
     ],
     install_requires=load_requirements('requirements.txt'),
-    tests_require=load_requirements('test-requirements.txt'),
+    tests_require=[
```

Comment:
This was legacy cargo cult. External requirement files were needed to install packages that weren't on PyPi (i.e. Github). All of the test packages are on PyPi, so we can remove the indirection. We don't gain anything from the extra file; `setup.py` is already just reading in its contents anyway.

`requirements.txt` is still installing XBlock via Github, so I'm considering that change outside the scope of this submission.